### PR TITLE
Add ebpf.io

### PR DESCRIPTION
**Pull request by zoidbergwill**
Closes https://github.com/zoidbergwill/awesome-ebpf/issues/35

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,7 +23,8 @@ BPF, as in _Berkeley Packet Filter_, is an in-kernel virtual machine running pro
 
 ### Cilium Guide
 
-- [Cilum's BPF and XDP Reference Guide](http://docs.cilium.io/en/latest/bpf/) - Generic documentation about most features of eBPF.
+- [Cilum's New BPF and XDP Reference Guide](https://ebpf.io/) - General documentation about features of eBPF, with a lot of pretty diagrams, and links to a bunch of related [projects](https://ebpf.io/projects).
```
[the remark this answers is not in the document]
Comment (zoidbergwill) [reply readme.md:26]: The Cilium section is the first section of links, but you're right maybe it belongs above the Table of Contents!

**Comment by zoidbergwill** [comment]
Thanks for the great feedback, @qmonnet and @pchaigno.

I moved it to the top as @pchaigno suggested, since ebpf.io does seem to be not too Cilium / Isovalent focused, though I think going with the great description @qmonnet suggested is a good idea if we moved it back.

Thank you both for helping contribute to this project.

File: readme.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 BPF, as in _Berkeley Packet Filter_, is an in-kernel virtual machine running programs passed from user space. Initially implemented on BSD, then Linux, the (now legacy) "classic BPF" or cBPF machine would be used with tools like tcpdump for filtering packets in the kernel to avoid useless copies to user space. More recently, the BPF infrastructure in Linux has been completely reworked and gave life to the "extended BPF", or eBPF, which gained new features (safety and termination checks, JIT-compiling for programs, persistent maps, a standard library, hardware offload support, etc.) and is now used for many tasks. Processing packets at a very low level (XDP), tracing and monitoring events on the system, or enforcing access control over cgroups are but a few examples to which eBPF brings performance, programmability and flexibility.
 
+Recently Cilium launched a great website about eBPF called [epf.io](https://ebpf.io/). It serves a similar purpose to this list, with a summary of eBPF and links to related [projects](https://ebpf.io/projects).
```
Comment (pchaigno) [inline readme.md:7]: Maybe:
```suggestion
Recently [Cilium](https://cilium.io) launched a great website about eBPF called [ebpf.io](https://ebpf.io/). It serves a similar purpose to this list, with [an introduction to eBPF](https://ebpf.io/what-is-ebpf) and links to [related projects](https://ebpf.io/projects).
```

**Comment by zoidbergwill** [comment]
Great suggestion!

**Review by pchaigno — APPROVED** [review]
LGTM!

**Comment by qmonnet** [comment]
Looks great! Should we _also_ add it to the list itself? Asking because I tend to often skip the intro and my eyes would jump directly to the bullet points, making it easy to skip the link. We could add it just above the Cilium guide, and rename the subsection (maybe `Cilium Guide` -> `Main Entry Points` or `eBPF-Specific Documentation` or ...?). But it really depends on what you prefer, I'm also fine with the current state of the PR.

**Comment by zoidbergwill** [comment]
Were you suggesting we move the `Cilium guide`'s reference documentation under the `eBPF-Specific Documentation` header too?

**Comment by qmonnet** [comment]
Yes, the item is named `Cilium's BPF and XDP Reference Guide`, so it already gives the information, and hardly needs a `Cilium Guide` section that we won't be able to reuse for any other link, I think?

**Comment by qmonnet** [comment]
Would be great to have this merged. @zoidbergwill I pushed a commit on top of yours, what do you think about it? Let's decide if we keep it or not and merge the result?

**Comment by zoidbergwill** [comment]
Whoops, my bad. Thought I'd updated the MR.

Your commit looks great! LGTM :+1:

**Pull request merged**